tasks/vg-default.yml

```diff
@@ -72,3 +72,33 @@
     - debug:
         msg: "Failed to wipe pv signatures."
   when: pool.type == 'lvm' and pool.state == "absent"
+
+#
+# Handle removing a partition when pool.state is absent and use_parititions is undefined/false
+#
+- name: Get the name of the pool's primary partition as applicable
```

Comment:
It would be more robust if you were to iterate over `pool._orig_members` instead and remove any that are partitions as there could be multiple partitions across several disks.

Reply:
Ended up iterating over `pool.disks` instead as throwing the partition path (e.g. `/dev/sdb1') to `parted` won't change anything and will evaluate to `ok: [localhost]`.